Process_Library/GOCART2G_Process.F90

```diff
@@ -1183,10 +1183,10 @@ subroutine Chem_SettlingSimple ( km, klid, flag, cdt, grav, &
 
    hsurf => hghte(i1:i2,j1:j2,km)
 
-   allocate(dz(i2,j2,km), radius(i2,j2,km), rhop(i2,j2,km), vsettle(i2,j2,km), qa(i2,j2,km))
+   allocate(dz(i2,j2,km), radius(i2,j2,km), rhop(i2,j2,km), vsettle(i2,j2,km), qa(i2,j2,km), source=0.0)
    allocate(cmass_before(i2,j2), cmass_after(i2,j2))
-   cmass_before = 0.d0
-   cmass_after = 0.d0
+   cmass_before = 0.0_DP
+   cmass_after = 0.0_DP
```

Comment:
[L1187-L1189](https://github.com/GEOS-ESM/GOCART/pull/131/files#diff-c654445ea9c7fe0dfe3d5bcf55254b11562955161b9315737ac00bfddba60ed4R1187-R1189) can be replaced with the equivalent: 
>  allocate(cmass_before(i2,j2), cmass_after(i2,j2), source=0.0_DP)

Reply:
Done. Please take a look